### ISSUE TITLE
Is the project still alive ?

### ISSUE THREAD

**Issue by PubliAlex** [GetmeUK/contenttools-2-roadmap#22]
Hello,

Following that project for quite some time, I just wonder if I have to consider it as "still alive" or is it dead ?

I'm asking because the readme talk about a release "no later that Jan 2021", there are an issue thank talk about alpha release access without official answer... It clearly seems that this repository has not been opened by maintainers since months.

If the answer is yes, it's still alive, is it possible to get a status update ?

Thank you for the answer

**Comment by anthonyjb** [comment]
Hi @PubliAlex,

We continue to support and use the existing CT in projects (in terms of bug fixes) and I am committed to developing a new release of CT in ES6 as part of our own tooling goals so this will happen because it's something we need internally. The version of CT we use already has various features that are new or replace existing CT features (such as the image editor which now supports named crops for different device sizes) all of which are rewritten in ES6. However, there's a lot still to do :/

Life has gotten in the way to a fairly large extent over the last 18 months for one reason or another and I continue to find only limited pockets of time on the project as other life and work projects take priority (had twins, moving house - yikes). 

So to summarise:

- The project isn't dead and is some way along in internal projects (but far from a release).
- I don't have a firm grasp on dates (and in hindsight I never should have set one).
- I realise this is far from ideal for anyone hoping to use CT as there's not road map for when the new release will be available.

**Comment by PubliAlex** [comment]
Hello,

Congratulation for your twins :) I perfectly understand what you mean cause I'm on a quite same situation, that's why I asked for a situation update, more than an alpha access.

So, I think it's better to announce a "when it's ready" date, instead of an irrealistic date in term of delay in your CT2 readme.

Best regards,

**Comment by ikkez** [comment]
Is there any chance that we, the open source community, can help you completing the rewrite to ES6? I just tried to use a coffeescript to ES6 Converter a while ago and actually that looked fine so far but I guess there's more about your refactoring than just the transition to ES6?!

**Comment by anthonyjb** [comment]
Hi @ikkez - I would really like to get help with the project from the OS community. 

I am looking to make some fundamental improvements to project as part of the rewrite (not just a ES6 rewrite), one of which is to improve the experience for other developers who want to contribute to the core project or those looking to extend it, the current coffeescript and weird package set up (I had no idea about NPM when I started CT) makes it really hard so starting with a better development pipeline would be a key initial goal.

The caveat here is that I don't have any experience managing such a process or even know where to begin with regard to organizing the project into chunks of work contributors can help with. At work I manage projects by creating tickets on a trello board but rarely manage the board for anyone but myself or perhaps 1-2 other developers. I'm not sure if that will translate to working on a project with an OS community - or even how big that community would be.

If anyone has contributed to other OS projects that are/were starting from very early stages (this is a rewrite rather than starting from scratch), had a really good experience and can recommend so that I could perhaps learn from there process I would be really grateful for the signpost.

**Comment by PubliAlex** [comment]
Hello,

The development of 2.x version of content tools seems to be completely stopped and the 1.x version not actively maintened. Is it possible to have an update of the project globally to see if it's still a solid library to use or if it's slowly dying and it's preferable to find (or create) something else to handle frontend editing.

Thank you
